### PR TITLE
Hub: Fixes summary of a resource on the resource card

### PR DESCRIPTION
**Pull request by pratap0007**
# Changes
- Previously,summary was missing for few resources on resource card.
- This patch adds missing summary for resources.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/experimental/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: hub/ui/src/components/task/Task.tsx [hunk 1]
```diff
@@ -190,7 +197,7 @@ const Task: React.FC<TaskProp> = (props: any) => {
                     <Badge style={{
                       marginLeft: '0.2em',
                       marginBottom: '1em',
-                    }} key={`badge-${tag}`} className="badge">{tag}</Badge>
+                    }} key={`badge-${ tag }`} className="badge">{tag}</Badge>
```
Comment (PuneetPunamiya) [inline hub/ui/src/components/task/Task.tsx:200]: Can we fix the trim spaces here ?

File: hub/ui/src/components/task/Task.tsx [hunk 2]
```diff
@@ -118,6 +121,23 @@ const Task: React.FC<TaskProp> = (props: any) => {
       return str.toUpperCase();
     }));
   }
+  // resource summary
+  let resourceSummary = '';
+  const summaryLength = props.task.description.length;
+  const summaryIndex = props.task.description.indexOf('\n');
+  if (summaryLength > 120) {
```
Comment (PuneetPunamiya) [inline hub/ui/src/components/task/Task.tsx:128]: Do we need the variable `summaryLength` instead can't we just directly use `props.task.description.length`

Comment (pratap0007) [reply hub/ui/src/components/task/Task.tsx:128]: yeah,I did it intentionally to avoid many time computation of props.task.description.length

Comment (pratap0007) [reply hub/ui/src/components/task/Task.tsx]: have removed variable I think computation would not be much overhead

**Pull request merged**